test/components/granules/overview.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import test from 'ava';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { shallow, configure } from 'enzyme';
+import { GranulesOverview } from '../../../app/scripts/components/granules/overview';
+
+configure({ adapter: new Adapter() });
+
+const granules = {
+  list: {
+    meta: {
+      count: 12,
+      queriedAt: 0
+    }
+  },
+  map: {
+    'my-granule-id': {
+      data: {
+        name: 'my-name',
+        filename: 'my-filename',
+        bucket: 'my-bucket',
+        status: 'success',
+        files: [
+          {
+            fileName: 'my-name',
+            key: 'my-key-path/my-name',
+            bucket: 'my-bucket'
+          }
+        ]
+      }
+    }
+  }
+};
+
+test('Overview renders bulkAction buttons with recovery button', function (t) {
+  const dispatch = () => {};
+  const workflowOptions = [];
+  const stats = { count: 0, histogram: {}, stats: {} };
+  const location = { pathname: 'granules' };
+  const config = { recoveryPath: 'recover' };
+  const store = {
+    subscribe: () => {},
+    dispatch: dispatch,
+    getState: () => {}
+  };
+
+  const providerWrapper = shallow(
+    <Provider store={store}>
```

Comment:
Since the `store` object above returns no state, is this doing anything?

Reply:
🤔 looking into it

Reply:
If I import `GranulesOverview` class, without the connect, from list-view.js, then I think this `{store}` isn't required. I'm having troubles getting this working without `{store}` passed into the provider - specifically, `dive()` into the `GranulesOverview` throws an error about "not having a store object passed in".

Reply:
This might be something that I can pass in as `dive({ store})`, but at that point it's half-a-dozen to 6.

Reply:
OK, I'm fine to leave this as is, but I'm a bit confused. If we are wrapping in `<Provider>` with the store, then we should be able to use the exported GranulesOverview with the connect? And so we shouldn't need the extra GranulesOverview export. Or is that not true?

Reply:
I just noticed that I'm doing the exact same thing for the reconciliation report tests. So this is 100% fine